Roommatch/Roommatch/Models/User.h

```diff
@@ -0,0 +1,32 @@
+//
+//  User.h
+//  Roommatch
+//
+//  Created by Lily Yang on 7/7/22.
+//
+
+#import <Parse/Parse.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface User : PFUser
+
+@property (strong, nonatomic) NSString* name;
+@property (strong, nonatomic) NSString* age;
+@property (strong, nonatomic) NSString* pronouns;
+@property (strong, nonatomic) NSString* priceLow;
+@property (strong, nonatomic) NSString* priceHigh;
+@property (strong, nonatomic) PFFileObject* profilePicture; 
+@property (strong, nonatomic) NSString* city;
+@property (strong, nonatomic) NSString* bio;
+@property (strong, nonatomic) NSString* smoking;
+@property (strong, nonatomic) NSString* pets;
+@property (strong, nonatomic) NSString* inCollege;
+@property (strong, nonatomic) NSString* collegeName;
+@property (strong, nonatomic) NSString* instagramTag;
+
```

Comment:
Some of the types above sound like it would make more sense as something other than a string.
Should `age` be an integer? Could the prices be doubles? Could `smoking` or `inCollege` be booleans?

Reply:
Since most of these are only being displayed, I wanted to leave things as a string. 
I wanted prices to be ints or doubles, but ran into errors when I tried to display the string value of the integer, so made them strings for now.